libs/Microsoft.MixedReality.WebRTC.Native/src/peer_connection.cpp

```diff
@@ -951,7 +951,8 @@ void PeerConnectionImpl::RemoveAllDataChannels() noexcept {
 
     // Invoke the DataChannelRemoved callback on the wrapper if any
     if (removed_cb) {
-      mrsDataChannelHandle data_native_handle = (void*)&data_channel;
+      DataChannel* const dc = data_channel.operator->();
```

Comment:
nit:
```suggestion
      DataChannel* const dc = data_channel.get();
```

Reply:
meh, IntelliSense refused it, but apparently `std::shared_ptr` does have a `get()` 😢

Reply:
Fixed.